virtual/activation_cache.go

```diff
@@ -83,16 +84,19 @@ func (a *activationsCache) ensureActivation(
 	moduleID,
 	actorID string,
 
-	blacklistedServerID string,
+	extraReplicas uint64,
```

Comment:
Can you file an issue to follow this P.R up with a new test in examples/leaderregistry that spawns some actors with more than 1 replica, then kills one of the servers (and thus 1 of the replicas for all actors) and ensures that requests can still succeed? I think with the current implementation ~50 of requests will fail (which is fine for now) and then once we have that test we can work on making it so requests continue to succeed with circuit breakers and retry policies etc

Reply:
#70